microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -558,48 +544,62 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 133.
-* This checkbox is present in Microsoft Edge Stable 131.
+* This checkbox is present in Microsoft Edge Canary 134.
+* This checkbox is present in Microsoft Edge Stable 132.
 
 
 <!-- ====================================================================== -->
-## Extension storage in Application panel
+## Floating entry points for the AI assistance panel
 <!-- checkbox has no "(?)" link pointing to this anchor wording -->
+<!-- todo: delete section if/when checkbox removed -->
+<!-- consistently hide or reveal all sections that have the above comment -->
```

Comment:
instance 2: comment out or reveal such sections?

Reply:
This section should be hidden. I don't think we can make a rule to consistently hide or reveal the sections that don't have a (?) icon. We will need to continue making this a case by case decision.

Reply:
commented out section
removed line: html comment